ui/src/timeMachine/constants/queryBuilder.ts

```diff
@@ -3,28 +3,77 @@ import {WINDOW_PERIOD, OPTION_NAME} from 'src/variables/constants'
 export interface QueryFn {
   name: string
   flux: string
-  aggregate: boolean
 }
 
 export const FUNCTIONS: QueryFn[] = [
-  {name: 'mean', flux: `|> mean()`, aggregate: true},
-  {name: 'median', flux: '|> toFloat()\n  |> median()', aggregate: true},
-  {name: 'max', flux: '|> max()', aggregate: true},
-  {name: 'min', flux: '|> min()', aggregate: true},
-  {name: 'sum', flux: '|> sum()', aggregate: true},
+  {
+    name: 'mean',
+    flux: `|> aggregateWindow(every: ${OPTION_NAME}.${WINDOW_PERIOD}, fn: mean)`,
+  },
+  {
+    name: 'median',
+    // TODO: Update to use `aggregateWindow` once
+    // https://github.com/influxdata/flux/issues/1244 is resolved. The
+    // `toFloat` call should still be present in the updated version.
```

Comment:
maybe an issue instead of a todo here?

Reply:
sure how about both, in case someone isn't aware of the issue when they are looking at the code

https://github.com/influxdata/influxdb/issues/13806